<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GithubUrlTran</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_url">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>6</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Url：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_url">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>6</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_tran">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>60</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tran：</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_tran">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>62</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_go">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>go</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="CDNgroupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>120</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>CDN</string>
    </property>
    <widget class="QRadioButton" name="radioButton_jsdelivr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>jsdelivr</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">CDNGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_staticaly">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>staticaly</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">CDNGroup</string>
     </attribute>
    </widget>
    <widget class="QRadioButton" name="radioButton_githack">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>githack</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">CDNGroup</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="CDNGroup"/>
 </buttongroups>
</ui>
